_posts/ja/newsletters/2025-06-27-newsletter.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Bitcoin Optech Newsletter #360'
+permalink: /ja/newsletters/2025/06/27/
+name: 2025-06-27-newsletter-ja
+slug: 2025-06-27-newsletter-ja
+type: newsletter
+layout: newsletter
+lang: ja
+---
+今週のニュースレターでは、P2Pプロトコルのメッセージを使ったフルノードのフィンガープリンティングに関する研究と、
+BIP380のディスクリプター仕様におけるBIP32パスで`H`のサポートを削除する可能性についてのフィードバックについて掲載しています。
+また、Bitcoin Stack Exchangeで人気の質問とその回答のまとめや、
+新しいリリースとリリース候補の発表、人気のBitcoinインフラストラクチャソフトウェアの注目すべき更新など
+恒例のセクションも掲載しています。
+
+## ニュース
+
+- **`addr`メッセージを使用したノードのフィンガープリンティング:** Daniela Brozzoniは、
+  開発者のNaiyomaと共に行った研究についてDelving Bitcoinに[投稿しました][brozzoni addr]。
+  この研究では、ノードが送信する`addr`（アドレス）メッセージを使用して、
+  複数のネットワーク上で同じノードを識別することができました。
+  ノードは、P2Pプロトコルの`addr`（アドレス）メッセージをピアに送信することで、
+  他の潜在的なノードをアナウンスし、ピアが分散型のゴシップシステムを使用してお互いを見つけられるようにしています。
+  しかし、BrozzoniとNaiyomaは、個々のノード固有のアドレスメッセージの詳細を使ってフィンガープリンティングを行い、
+  複数のネットワーク（IPv4と[Tor][topic anonymity networks]など）上で動作する同一ノードを識別することに成功しました。
+
+  研究者らは、２つの可能な緩和策を提案しています: アドレスメッセージからタイムスタンプを削除するか、
+  タイムスタンプを残す場合は、特定のノードに固有にならないようにタイムスタンプをわずかにランダム化することです。
+
+- **ディスクリプターで`H`を使用しているソフトウェアはありますか？** Ava Chow は、
+  強化導出の[BIP32][topic bip32]鍵導出ステップを示すために大文字のHを使ってディスクリプターを生成するソフトウェアがあるかどうかを
+  Bitcoin-Devメーリングリストで[尋ねました][chow hard]。もしない場合は、
+  [アウトプットスクリプトディスクリプター][topic descriptors]の[BIP380][]仕様は、
+  強化導出を示すために小文字のhと`'`のみを使用するように変更される可能性があります。
+  Chowは、BIP32では大文字のHが許可されているものの、BIP380には以前、
+  大文字のHの使用を禁止するテストが含まれており、Bitcoin Coreは現在大文字のHを受け入れていないことを指摘しています。
+
+## Bitcoin Stack Exchangeから選ばれたQ&A
+
+*[Bitcoin Stack Exchange][bitcoin.se]はOptech Contributor達が疑問に対して答えを探しに（もしくは他のユーザーの質問に答える時間がある場合に）アクセスする、
+数少ない情報ソースです。この月刊セクションでは、前回アップデート以降にされた、最も票を集めた質問・回答を紹介しています。*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Bitcoin Knotsノードをピアとしてブロックする方法はありますか？]({{bse}}127456)
+  Vojtěch Strnadは、Bitcoin Coreの2つのRPCを使って
+  ユーザーエージェントの文字列に基づいてピアをブロックする方法を提供していますが、
+  この方法は推奨されておらず、同様の非推奨事項を指摘する関連する[Bitcoin CoreのGitHubの課題][Bitcoin Core #30036]も示されています。
+
+- [OP_CATは整数をどのように処理しますか？]({{bse}}127436)
+  Pieter Wuilleは、Bitcoin Scriptのスタック要素にはデータの型情報が含まれておらず、
+  opcodeによってスタック要素のbyteの解釈方法が異なると指摘しています。
+
+- [コンパクトブロックリレーによる非同期ブロックリレー（BIP152）について]({{bse}}127420)
+  ユーザーbca-0353f40eは、Bitcoin Coreの[コンパクトブロック][topic compact block relay]の処理を概説し、
+  欠落したトランザクションがブロックの伝播に与える影響を推定しています。
+
+- [セルフィッシュマイニングにおける攻撃者の収益がハッシュパワーに不相応なのはなぜですか？]({{bse}}53030)
```

Comment:
auto linking failing here with just a "#"

Reply:
oh, I fixed it, thanks!